relayer-cli/src/utils/proof.ts

```diff
@@ -5,7 +5,7 @@ const getMessageDataToRelay = async (chainid: number, nonce: number) => {
     const subgraph = getSubgraph(chainid);
 
     const result = await request(
-      `https://api.studio.thegraph.com/query/67213/${subgraph}/version/latest`,
+      `https://api.studio.thegraph.com/query/85918/${subgraph}/version/latest`,
```

Comment:
Consider extracting this into an environment variable, if not the whole prefix at least the magic number

Reply:
Done in [commit](https://github.com/kleros/vea/pull/305/commits/f01e4f920d5eb4179c069c41c583713e75d74709).